sentences/sv/_common.yaml

```diff
@@ -52,14 +52,13 @@ expansion_rules:
   vad: "(vad är | vad är det | vad är det för | vilken)"
   är: "(är det | är)"
   brightness: "{brightness} [percent]"
-  slå_på: "( tänd | sätt på | slå på | starta )"
-  slå_av: "( släck | stäng av | stoppa )"
+  slå_på: "( tänd | sätt (på |igång | fart på)   | slå (på | till)  | starta )"
```

Comment:
There is some additional spaces in the expansion_rule that can be removed.

Reply:
fixed :)